client/src/components/EventSummary.tsx

```diff
@@ -310,7 +338,7 @@ const EventSummaryRow = ({ event, showEventSeries }: EventSummaryRowProps) => {
           <Col md={12}>
             <span>
               <strong>{Settings.fields.event.people.label}:</strong>{" "}
-              {event.people.map((person, i) => (
+              {sortedAttendees.map((person, i) => (
```

Comment:
You should also give some sort of indication (e.g. at the end of the list) when there are more than `PEOPLE_ATTENDING_LIMIT` attending people.